api/datastore/sql/sql.go

```diff
@@ -43,50 +40,6 @@ import (
 // we'll get better locality with varchar. it's not terribly easy to do this
 // with migrations (sadly, need complex transaction)
 
-var tables = [...]string{`CREATE TABLE IF NOT EXISTS routes (
-	app_name varchar(256) NOT NULL,
-	path varchar(256) NOT NULL,
-	image varchar(256) NOT NULL,
-	format varchar(16) NOT NULL,
-	memory int NOT NULL,
-	cpus int,
-	timeout int NOT NULL,
-	idle_timeout int NOT NULL,
-	type varchar(16) NOT NULL,
-	headers text NOT NULL,
-	config text NOT NULL,
-	created_at text,
-	updated_at varchar(256),
-	PRIMARY KEY (app_name, path)
-);`,
-
-	`CREATE TABLE IF NOT EXISTS apps (
-	name varchar(256) NOT NULL PRIMARY KEY,
-	config text NOT NULL,
-	created_at varchar(256),
-	updated_at varchar(256)
-);`,
-
-	`CREATE TABLE IF NOT EXISTS calls (
-	created_at varchar(256) NOT NULL,
-	started_at varchar(256) NOT NULL,
-	completed_at varchar(256) NOT NULL,
-	status varchar(256) NOT NULL,
-	id varchar(256) NOT NULL,
-	app_name varchar(256) NOT NULL,
-	path varchar(256) NOT NULL,
-	stats text,
-	error text,
-	PRIMARY KEY (id)
-);`,
-
-	`CREATE TABLE IF NOT EXISTS logs (
-	id varchar(256) NOT NULL PRIMARY KEY,
-	app_name varchar(256) NOT NULL,
-	log text NOT NULL
-);`,
-}
-
```

Comment:
i don't see any code to replace these... we still need to handle fresh dbs.

Reply:
That is handled inside the migration. Fresh DB means that it doesn't have the migrations applied, so the migration DB check will return `-1` as the indicator of a fresh new datastore.

Reply:
right, but I don't see the `CREATE TABLE` replacements ? maybe github didn't expand the new file they are in and my search is failing?

Reply:
it's there, at `migration.so`, no worries.